x-pack/auditbeat/module/system/packages/packages.go

```diff
@@ -2,6 +2,8 @@
 // or more contributor license agreements. Licensed under the Elastic License;
 // you may not use this file except in compliance with the Elastic License.
 
+// +build ignore
```

Comment:
Rather than build tags to ignore it how about commenting out the code that registers the metricset. This will help prevent code rot because the code will still be subject to compiling and go unit testing, but won’t be available to use.

Reply:
Makes sense, I've changed it.